apps/i18n/common/en_us.json

```diff
@@ -460,6 +460,8 @@
   "online": "Online",
   "openWorkspace": "How It Works",
   "orientationLock": "Turn off orientation lock in device settings.",
+  "onetimeHintPromptTitle": "Feeling Stuck?",
+  "onetimeHintPromptMessage": "Click the lightbulb for a hint",
```

Comment:
@ryansloan there's still an open question in the spec if this is the copy we actually want to go with

Reply:
Go with this copy for now. We are also hoping to help discoverability of the lightbulb with this so I think explicitly spelling it out is good to start.